deployment/gke/README.md

```diff
@@ -0,0 +1,75 @@
+# Deploy process
+
+There are 3 region clusters each with 3 regions (9 nodes), and each node is a
+e2-highmem-4 machine type. The region are: `asia-east1`, `europe-west1`, `us-west1`
+
+## Build Docker image and push to Cloud Container Registry
+
+```bash
+cd ../
```

Comment:
it would be nice to turn these steps into a script once the dust settles

Reply:
Moved them into scripts (also to ease my testing..)